README.md

```diff
@@ -658,11 +658,11 @@ To use VSCode command 'Increase/decrease current view size' instead of separate
 
 ### Insert mode special keys
 
-Enabled by `useCtrlKeysForInsertMode` (default true).
+Enabled by `ctrlKeysForInsertMode` (default `["a", "d", "h", "j", "o", "r", "t", "u", "w"]`).
 
 Refer to VIM's manual for their use.
 
--   <kbd>C-c</kbd>
+-   <kbd>C-c</kbd> (bind to command `vscode-neovim.escape`)
```

Comment:
I think this is a good chance to delete this list, it doesn't serve a real purpose.